### ISSUE TITLE
is any dark mode available?

### ISSUE THREAD

**Issue by plainee** [ProgerXP/Notepad2e#373]
is any dark mode available?

**Comment by ProgerXP** [comment]
Nope, and there are no such plans for now. Supporting dark mode needs implementing themes in Scintilla and it might be a big task (haven't checked though). Are you using Notepad 2e a lot? (on Arch?..)
